_dev/src/components/smart-shopping-campaign/reporting/campaigns-performance/campaigns-performance-table-row.vue

```diff
@@ -3,12 +3,21 @@
     <b-td
       class="b-table-sticky-column text-primary"
     >
-      <b-button
+      <div
+        class="p-0 m-0 font-weight-normal ps_gs-fz-12"
```

Comment:
I wonder if we can make the feature ready immediately, with a 

`v-if="campaign.id"`